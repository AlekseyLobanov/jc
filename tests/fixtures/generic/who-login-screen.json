[{"user":"atemu","tty":"seat0","time":"2023-10-21 20:06","from":"login screen","epoch":1697943960},{"user":"atemu","tty":":0","time":"2023-10-21 20:06","from":":0","epoch":1697943960},{"user":"atemu","tty":"pts/8","time":"2023-10-23 18:27","epoch":1698110820}]
