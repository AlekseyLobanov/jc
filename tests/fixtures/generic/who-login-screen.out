atemu    seat0        2023-10-21 20:06 (login screen)
atemu    :0           2023-10-21 20:06 (:0)
atemu    pts/8        2023-10-23 18:27
